src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -86,12 +88,28 @@ public function getSchema(): Schema
             $graphqlConfiguration = $resourceMetadata->getGraphql() ?? [];
             foreach ($graphqlConfiguration as $operationName => $value) {
                 if ('query' === $operationName) {
-                    $queryFields += $this->getQueryFields($resourceClass, $resourceMetadata);
+                    $queryFields += $this->getQueryFields($resourceClass, $resourceMetadata, $operationName, ['args' => ['id' => ['type' => GraphQLType::id()]]], true);
 
                     continue;
                 }
 
-                $mutationFields[$operationName.$resourceMetadata->getShortName()] = $this->getMutationFields($resourceClass, $resourceMetadata, $operationName);
+                if ($item = $resourceMetadata->getGraphqlAttribute($operationName, 'item_query')) {
+                    $value['resolve'] = $this->queryResolverLocator->get($item);
+                    unset($value['item_query']);
+
+                    $queryFields += $this->getQueryFields($resourceClass, $resourceMetadata, $operationName, $value, false);
+                    continue;
+                }
+
+                if ($collection = $resourceMetadata->getGraphqlAttribute($operationName, 'collection_query')) {
```

Comment:
```suggestion
                if ($collectionConfiguration = $resourceMetadata->getGraphqlAttribute($operationName, 'collection_query')) {
```

Reply:
`collection_query` is not the same as `$collectionConfiguration`